{
    "_type": "pyessv._model.term", 
    "canonicalName": "qc-flag-channel-6-failure", 
    "createDate": "2018-03-09 00:00:00+00:00", 
    "data": {
        "dimension": "time", 
        "flagMeanings": "not_used good_data  bad_data_channel_failure suspect_data_time_stamp_error", 
        "flagValues": "0b,1b, 2b, 3b", 
        "longName": "Data Quality flag: Receiver channel 6 failed", 
        "name": "qc_flag_channel_6_failure", 
        "type": "byte", 
        "units": "1"
    }, 
    "idx": 4, 
    "label": "qc_flag_channel_6_failure", 
    "rawName": "qc_flag_channel_6_failure", 
    "status": "pending", 
    "uid": "857bb9d6-c2ae-47d5-8e7c-6a6278f93a54"
}